### PR TITLE
Make timestamp type safe

### PR DESCRIPTION
**Pull request by i-schuetz**
This helps to ensure that we're actually using Unix time everywhere.

I considered not exposing the fact that the internal value is Unix time, but I think this would make the solution too complicated and possibly affect performance, as we'd not be able to operate externally with the timestamps.

### CODE REVIEW

**Comment by i-schuetz** [comment]
Actually it probably makes more sense to call this class `UnixTime`. `CoEpiDate` would be meaningful if we wanted to change the internal representation, but this would break e.g. the database and it's not hiding it anyway.

File: CoEpi/domain/model/UnixTime.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+struct UnixTime: Codable {
+    let value: Int64
+
+    static func fromUnixTime(unixTime: Int64) -> UnixTime {
```
Comment (duskoo) [inline CoEpi/domain/model/UnixTime.swift:6]: Confusing: method creates UnixTime object from Int64.  Default initializer can be used instead.

Comment (i-schuetz) [reply CoEpi/domain/model/UnixTime.swift:6]: Right, forgot to update this.

File: CoEpi/domain/model/UnixTime.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+struct UnixTime: Codable {
+    let value: Int64
+
+    static func minTimestamp() -> UnixTime {
+        UnixTime(value: 0)
+    }
+
+    static func now() -> UnixTime {
+        UnixTime(value: Int64(Date().timeIntervalSince1970))
```
Comment (duskoo) [inline CoEpi/domain/model/UnixTime.swift:11]: @i-schuetz This might be a good point to switch to millisecond resolution that V4 backend is expecting.

Comment (i-schuetz) [reply CoEpi/domain/model/UnixTime.swift:11]: There has not been a final decision on the backend resolution yet.

**Pull request merged**